pytest-selenium/pages/content.py

```diff
@@ -1,21 +1,36 @@
 from selenium.common.exceptions import NoSuchElementException
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions as expected
+from selenium.webdriver.common.action_chains import ActionChains
 
 from pages.base import Page
 from regions.base import Region
 
 
 class Content(Page):
     URL_TEMPLATE = "/books/{book_slug}/pages/{page_slug}"
-
     _body_locator = (By.TAG_NAME, "body")
     _main_content_locator = (By.CSS_SELECTOR, "h1")
+    _next_locator = (By.CSS_SELECTOR, "[aria-label='Next Page']")
+    _previous_locator = (By.CSS_SELECTOR, "[aria-label='Previous Page']")
+    _scroll_lock_locator = (By.CSS_SELECTOR, "[class*='MobileScrollLock']")
 
     @property
     def loaded(self):
         return self.find_element(*self._body_locator).get_attribute("data-rex-loaded")
 
+    @property
+    def next_link(self):
+        return self.find_element(*self._next_locator)
+
+    @property
+    def previous_link(self):
+        return self.find_element(*self._previous_locator)
+
+    @property
+    def is_scroll_locked(self):
+        return self.find_element(*self._scroll_lock_locator)
```

Comment:
is there no way to actually try to scroll it? just looking for a css class isn't really testing this

Reply:
@TomWoodward I can try capturing the exception that raises, similar to verifying the disabled state of the elements

Reply:
@TomWoodward with latest commit, I have used scroll_from_element() and captured the exception in assert statement.
I tried scroll(x,y) but that would not throw any exception. 
Let me know if you think some other methods can be used more effectively here.

Reply:
@TomWoodward updated the test to use the random click as mentioned above